corehq/apps/users/templates/users/roles_and_permissions.html

```diff
@@ -123,6 +123,11 @@ <h3 class="panel-title">
             {% trans "Data Registries" %}
           </th>
           {% endif %}
+          {% if request|toggle_enabled:"ATTENDANCE_TRACKING" %}
```

Comment:
I think Attendance Tracking has both a privilege and a (temporary) toggle associated with it.
```suggestion
          {% if request|request_has_privilege:"ATTENDANCE_TRACKING" and request|toggle_enabled:"ATTENDANCE_TRACKING" %}
```